### ISSUE TITLE
Timetable

### ISSUE THREAD

**Issue by Alexxiioo** [edqx/node-smhw-client#1]
Was there any way to access the timetable through the API, such as what lessons are set for tomorrow?

**Comment by edqx** [comment]
Nope, Timetables are a separate service to showmyhomework that I don't have, so I can't test it to work on on it, sorry. If you plan to use this library, I made a newer https://github.com/edqx/node-smhw instead.
